include/zephyr/net/net_pkt.h

```diff
@@ -194,6 +194,9 @@ struct net_pkt {
 	uint8_t l2_processed : 1; /* Set to 1 if this packet has already been
 				   * processed by the L2
 				   */
+	uint8_t chksum_done : 1; /* L4 Checksum has already been computed for
```

Comment:
This is used by L3 (IP) as well, no? also if you want abbreviate `csum` might be better?

Reply:
I think we can drop L4 here, we either calculate either both (L3/L4) checksums or none anyway.
As for the abbreviation, I'd prefer to stick to `chksum`, as it's already widely used in Zephyr, in checksum related routines.